src/BeaconChain.sol

```diff
@@ -60,8 +65,12 @@ contract BeaconChain {
     Queue[] public depositQueue;
     Queue[] public withdrawQueue;
     Validator[] public validators; // unordered list of validator
+
+    uint256 public uniqueDepositId = 100; // unique identifier for deposits, starting at 100 to avoid confusion with 0
```

Comment:
You can put this logic into the `DepositContract.sol` and auto increase anytime the `function Deposit` is called